src/publishAsset.ts

```diff
@@ -44,13 +44,7 @@ export async function publishAsset(params: PublishAssetParams, signer: Signer, c
         license: 'MIT',
         tags: params.tags
       },
-      stats: {
```

Comment:
we turned back the `Asset` into a `DDO`, removing the stats..
but by removing this lines:
```
price: {
          value: params.isCharged ? params.price : "0"
        }
```
we now have a FixedPrice datatoken instead of  Dispenser one
We totally changed the behavior of the previous function
Before it was doing:
` bundleNFT = await nftFactory.createNftWithDatatokenWithDispenser(`
Now it does always
`bundleNFT = await nftFactory.createNftWithDatatokenWithFixedRate(`